inductiva/_cli/cmd_logs/logs.py

```diff
@@ -1,22 +1,86 @@
 """CLI for logs."""
+import json
+import re
 import sys
+from typing import Tuple
 
 from .. import utils as cli_utils
-from ... import tasks
+from ... import tasks, ApiException
 
 
-def stream_task_logs(args):
-    """Consume the stream logs of a certain task."""
-    task_id = args.task_id
-    task = tasks.Task(task_id)
+def _get_task_id_from_mode(mode: str) -> Tuple[bool, str]:
+    """Extract the task_id from the mode."""
+    match = re.match(r"^(submitted|started)(-(\d+))?$", mode)
+    if not match:
+        return False, f"Invalid mode format: {mode}"
+
+    status_match = match.group(1)
```

Comment:
I think this variable could be just `status`.

Reply:
Ops, yeah nice catch!
thanks